packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -46,6 +46,8 @@ class DebuggerController extends DisposableController
     }();
   }
 
+  ValueNotifier get hasFrames => _hasFrames;
```

Comment:
nit: place this getter right next to _hasFrames

Reply:
will do